### PR TITLE
New pakage: libexodus

### PR DESCRIPTION
**Pull request by 3rav**

### CODE REVIEW

File: mingw-w64-libexodus/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Maintainer: Rafał Brzegowy <rafal.brzegowy@yahoo.com>
+
+_realname=libexodus
+_releases=2020-08-13
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=8.07
```
Comment (lazka) [inline mingw-w64-libexodus/PKGBUILD:7]: Where is this version from?

File: mingw-w64-libexodus/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+pkgrel=1
+pkgdesc="EXODUS is a model developed to store and retrieve data for finite element analyses (mingw-w64)"
+arch=('any')
+depends=("${MINGW_PACKAGE_PREFIX}-crt")
+makedepends=("unzip"
+             "${MINGW_PACKAGE_PREFIX}-netcdf")
+url="https://gsjaardema.github.io/seacas/"
+license=('BSD')
+source=("https://github.com/gsjaardema/seacas/archive/v${_releases}.zip")
```
Comment (lazka) [inline mingw-w64-libexodus/PKGBUILD:16]: What about using the tar.gz variant?

File: mingw-w64-libexodus/PKGBUILD (hunk 1 above)
Comment (3rav) [reply mingw-w64-libexodus/PKGBUILD:7]: Hi,

From exodusII.h:
```
/* EXODUS version number */
#define EXODUS_VERSION "8.07"
#define EXODUS_VERSION_MAJOR 8
#define EXODUS_VERSION_MINOR 07
#define EXODUS_RELEASE_DATE "June 16, 2020"
```

File: mingw-w64-libexodus/PKGBUILD (hunk 2 above)
Comment (3rav) [reply mingw-w64-libexodus/PKGBUILD:16]: bsdtar could not unpack zip and tar.gz  archive

File: mingw-w64-libexodus/PKGBUILD (hunk 1 above)
Comment (lazka) [reply mingw-w64-libexodus/PKGBUILD:7]: ok, thanks!

File: mingw-w64-libexodus/PKGBUILD (hunk 2 above)
Comment (lazka) [reply mingw-w64-libexodus/PKGBUILD:16]: ok then, thanks

**Pull request merged**